doc/manual/55-barman-cli.en.md

```diff
@@ -73,6 +73,21 @@ and can be installed alongside the PostgreSQL server:
 - `barman-cloud-restore`: script to be used to restore a backup directly
   taken with `barman-cloud-backup` from cloud storage;
 
+These commands require the appropriate library for the cloud provider you wish to
+use:
+
+* AWS S3: [boto3][boto3]
+* Azure Blob Storage: [azure-storage-blob][azure-storage-blob] and (optionally)
+  [azure-identity][azure-identity]
+* Google Cloud Storage: [google-cloud-storage][google-cloud-storage]
+
+**NOTE:** If you are using the Barman cloud utilities with Python 2 then you will
+need to ensure the following version requirements are met:
```

Comment:
I would add some strong recomandation to move to python3 because of python2 end of support coming. If it is not already somewhere.

Reply:
Done